wordpress-mobile/WordPress-Android
# Fixes: My site dashboard menu items tracking

Fixes #19114

## Description
This PR implements the tracking on clicking menu items in MySiteDashboard

## To test:
1. Login to the app
2. Go to Menu
3. Tap on menu item posts|stats
4. Verify that the tracking is present
5. 🔵 Tracked: my_site_menu_item_tapped, Properties: {"type":"posts|stats"}
5. Tap on any other item
6. Verify that following tracking is present
7. 🔵 Tracked: my_site_menu_item_tapped, Properties: {"type":"MENU_LABEL"}

## Regression Notes
1. Potential unintended areas of impact
Incorrect tracking 

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing and Unit testing

3. What automated tests I added (or what prevented me from doing so)
Unit tests

PR submission checklist:

UI Changes testing checklist:
